op-e2e/actions/proofs/operator_fee_test.go

```diff
@@ -234,9 +234,6 @@ func Test_ProgramAction_OperatorFeeConsistency(gt *testing.T) {
 			env.Batcher.ActL2BatchBuffer(t)
 
 			aliceInitialBalance, l1FeeVaultInitialBalance, baseFeeVaultInitialBalance, sequencerFeeVaultInitialBalance, operatorFeeVaultInitialBalance = getCurrentBalances()
-			if testCfg.Custom != NotEnoughFundsInBatchMissingOpFee {
-				require.Equal(t, operatorFeeVaultInitialBalance.Sign(), 0)
```

Comment:
instead of removing this check, shouldn't we add it to all the other cases after the initial `getCurrentBalances()` calls?